.github/workflows/docker_buildx-on-tag-publish.yml

```
@@ -0,0 +1,56 @@
+        uses: actions/checkout@v3
+
+      - name: Setup please.build
+        uses: sagikazarmark/setup-please-action@v0
+      
+      - name: Extract app name and version from tag
+        id: extract-app-name-version
+        run: |
+          echo "Tag: $GITHUB_REF"
+          APP_NAME=$(echo "$GITHUB_REF" | cut -d '/' -f 1)
+          VERSION=$(echo "$GITHUB_REF" | cut -d '/' -f 2)
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)